crates/levm/src/vm.rs

```diff
@@ -17,6 +27,12 @@ fn negate(value: U256) -> U256 {
     !value + U256::one()
 }
 
+fn address_to_word(address: Address) -> U256 {
+    let mut word = [0; 32];
+    word[12..].copy_from_slice(address.0.as_slice());
+    U256::from_big_endian(&word)
+}
```

Comment:
Nit. This must import `FromStr` -> `use std::str::FromStr;`. An `Address` has 20 bytes and an `U256` 32 bytes so an `Address` will always fit into a `U256`.

```suggestion
fn address_to_word(address: Address) -> U256 {
    U256::from_str(&format("{address:?}"));
}
```